README.md

```diff
@@ -47,6 +47,11 @@ const CheckoutForm = () => {
 
   const handleSubmit = async (event) => {
     event.preventDefault();
+
+    if (elements == null) {
```

Comment:
I think it will be better UI experience if you change line 64 to say `disabled={!stripe || !elements}`, so that the button is not clickable until elements becomes non-null. Then it would be safe to use `elements` in the handleSubmit.